lib/web/apiserver_test.go

```diff
@@ -3965,6 +3967,137 @@ func TestChangeUserAuthentication_WithPrivacyPolicyEnabledError(t *testing.T) {
 	require.True(t, apiRes.PrivateKeyPolicyEnabled)
 }
 
+func TestChangeUserAuthentication_settingDefaultClusterAuthPreference(t *testing.T) {
+
+	tt := []struct {
+		cloud               bool
+		name                string
```

Comment:
nit: name first, so the test scenario is the first information we see.

Ditto for the structs below.

```suggestion
		name                string
		cloud               bool
```